pyinstaller-Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM debian:stretch
```

Comment:
Not a big deal, but can you rename the file to Dockerfile.pyinstaller so it matches the pattern we have for all other dockerfiles (like Dockerfile.ci)?

Reply:
sounds appropriate! thanks for pointing it out